### ISSUE TITLE
show issue in channel (unfurled)

### ISSUE THREAD

**Issue by brianleroux** [smallwins/bugbot#47]
feature to show an issue in channel unfurtled

**Comment by jonotron** [comment]
Yup. Public repo issues "unfurl" ("unfurtle"??) but private repos issues do not. Since you already have my auth token you could grab the necessary details.

Possible security concern: I could accidentally link to a private repo that members of the slack channel should not see.

**Comment by nickbouton** [comment]
@brianleroux +1 -- was coming over here to suggest this, actually. Would be great if there were a way to reference an issue number in a message and at least have it link to the original issue & show some basic metadata (title, first line or two of the description, assignee, date created) underneath as an attachment.
